content/zh/docs/topics/registries.md

```diff
@@ -217,6 +217,6 @@ $ cat ~/Library/Caches/helm/registry/cache/blobs/sha256/31fb454efb3c69fafe536725
 
 ## 从chart仓库迁移
 
-从经典 [chart 仓库](https://helm.sh/docs/topics/chart_repository.md)
+从经典 [chart 仓库](https://docs.helm.sh/zh/docs/topics/chart_repository.md)
```

Comment:
The correct URL will be https://helm.sh/zh/docs/topics/chart_repository/ although you can just say `/docs/topics/chart_repository/` too.